client/clientutil/config.go

```diff
@@ -52,6 +53,10 @@ func ParseConfigValues(confValues []string, opts *ParseConfigOptions) (map[strin
 		parts := strings.SplitN(patchValue, "=", 2)
 		if len(parts) == 1 && strings.HasSuffix(patchValue, "!") {
 			key := strings.TrimSuffix(patchValue, "!")
+			if key == "" {
+				return nil, nil, errors.New(i18n.G("configuration keys cannot be empty"))
```

Comment:
maybe we could use a more specific error for `!` input:
```suggestion
				return nil, nil, errors.New(i18n.G("configuration keys cannot be empty (want key! to unset a key)"))
```

Reply:
Done, thanks!